lib/brakeman/app_tree.rb

```diff
@@ -96,13 +128,28 @@ def select_files(paths)
 
     def select_only_files(paths)
       return paths unless @only_files
-      paths.select { |f| @only_files.match f }
+      project_root  = Pathname.new(@root)
+      paths.select do |path|
+        absolute_path = Pathname.new(path)
+        # relative root never has a leading separator. But, we use a leading
+        # separator in a @skip_files entry to imply that a directory is
+        # "absolute" with respect to the project directory.
+        project_relative_path = File::SEPARATOR + absolute_path.relative_path_from(project_root).to_s
```

Comment:
Updated in https://github.com/presidentbeef/brakeman/commit/69057b386b7d717a21a1c50902943b927c27e078.